### ISSUE TITLE
Preferences not showing up

### ISSUE THREAD

**Issue by bgedik** [kylehuff/webpg-firefox#12]
My preferences pane is not showing the contents properly. I am on a Mac using Firefox 21. Attached is a screenshot.

![screenshot](https://f.cloud.github.com/assets/380854/643932/f37b9702-d380-11e2-9588-22265d68dd1c.png)

**Comment by kylehuff** [comment]
This is caused by an issue with Firefox on OSX, there are only two ways to get around this issue.

1.) Install the development version of WebPG, which introduces a workaround to this problem. (Instructions for installing the current development cycle are detailed here: https://github.com/kylehuff/webpg-chrome/wiki#install-development-build )

OR

2.) Disable a particular option within firefox (animateFadeIn) by opening Firefox, enter the URL "about:config" and enter "animateFadeIn" into the search box, then toggle the value of that preference from "true" to "false" by double-clicking the preference line.

Unfortunately, those are the only two ways to get around the issue that I am aware of.
NOTE: Only one of those methods are required. Either install version v0.9.4 (or above), OR disable the animateFadeIn option.

**Comment by bgedik** [comment]
Your ./build.sh script does not work in mac.

**Comment by kylehuff** [comment]
Yes, I forgot that the build script is in limbo at the moment; I will update the wiki to link to the development build XPI for OSX (http://webpg.org/download/webpg-firefox/webpg-firefox-devel.xpi)

**Comment by indolering** [comment]
Then update the Mozilla Add-Ons site to reflect that it is not working for 21 or backport a fix.  You are (rightfully) getting bad reviews on AMO.

**Comment by kylehuff** [comment]
No. It is not a bug within WebPG, it is a bug within Firefox on OSX. My caveat emptor is pretty clear that this is beta software and there is an expectation that some things will not work as intended within some configurations/systems.
